parquet/src/main/java/com/netflix/iceberg/parquet/ParquetMetrics.java

```diff
@@ -86,9 +86,9 @@ public static Metrics fromMetadata(ParquetMetadata metadata) {
           Types.NestedField field = fileSchema.asStruct().field(fieldId);
           if (field != null && stats.hasNonNullValue()) {
             updateMin(lowerBounds, fieldId,
-                fromParquetPrimitive(field.type(), stats.genericGetMin()));
+              fromParquetPrimitive(field.type(), column.getPrimitiveType(), stats.genericGetMin()));
```

Comment:
Nit: continuation lines should be indented 4 spaces from the start of the statement.

Reply:
Would no longer fit into 100 characters per line. Do we have such a requirement? I've seen a couple of places where this is not respected.

Reply:
That's probably a mistake then. Thanks for fixing this.